ui-frontend/packages/catalog-ui-search/src/main/webapp/component/visualization/maps/cesium/bbox-display.tsx

```diff
@@ -196,12 +266,58 @@ const useListenToModel = ({ model, map }: { model: any; map: any }) => {
   }, [callback])
 }
 
-export const CesiumBboxDisplay = ({ map, model }: { map: any; model: any }) => {
-  useListenToModel({ map, model })
+const useStartMapDrawing = ({
+  map,
+  model,
+  setNewBbox,
+  onDraw,
+}: {
+  map: any
+  model: any
+  setNewBbox: (newBbox: BBox) => void
+  onDraw: (newBbox: BBox) => void
+}) => {
+  React.useEffect(() => {
+    if (map && model) {
+      map.getMap().drawHelper[`startDrawingExtent`]({
+        callback: (extent: Cesium.Rectangle) => {
+          const bbox = rectangleToBbox(extent)
+          setNewBbox(bbox)
+          onDraw(bbox)
+        },
+        material: Cesium.Material.fromType('Color', {
+          color: Cesium.Color.fromAlpha(Cesium.Color.BLUE, 0.2),
+        }),
+      })
+    }
+    return () => {}
```

Comment:
❓  Is this empty return needed?

Reply:
No. If our effect doesn't need any cleanup, then we don't have to return a function. I'll remove these.